docs/pages/integrating/api.rst

```diff
@@ -21,6 +21,7 @@ Common Schemas and Responses: `SSI types <../../_static/common/ssi_types.yaml>`_
                     {url: "../../_static/monitoring/v1.yaml", name: "Monitoring"},
                     {url: "../../_static/vcr/vcr_v2.yaml", name: "Verifiable Credential Registry (v2)"},
                     {url: "../../_static/vdr/v2.yaml", name: "Verifiable Data Registry (v2)"},
+					{url: "../../_static/policy/v1.yaml", name: "Policy backend"},
```

Comment:
Indenting is off. Also not a callable Nuts node API, so I'd leave it out here.

Reply:
move this swagger part to policy page?

Reply:
Yeah, I'd say just add a link to it?